analyses/cell-type-ewings/README.md

```diff
@@ -32,6 +34,62 @@ This module will include code to annotate cell types in the Ewing sarcoma sample
+
+Defining tumor cells: [`AUCell`](https://bioconductor.org/packages/release/bioc/html/AUCell.html) was run on the merged object containing all libraries with a set of MSigDB and custom gene sets specific to Ewing sarcoma using the `run-aucell-ews-signatures.sh` workflow.
+See [below for detailed instructions on running this workflow](#using-aucell-to-calculate-gene-set-signatures). 
+
+Tumor cells were classified as those that met the following criteria: 
+
+- `tumor EWS-high`: 
+  - AUC > 0.04 for [`aynaud-ews-targets`](./references/gene_signatures/aynaud-ews-targets.tsv)
+  - AUC > 0.01 for [`STAEGE_EWING_FAMILY_TUMOR`](https://www.gsea-msigdb.org/gsea/msigdb/human/geneset/STAEGE_EWING_FAMILY_TUMOR.html)
+- `tumor EWS-low`: 
+  - AUC > 0.1 for [`wrenn-nt5e-genes`](./references/gene_signatures/wrenn-nt5e-genes.tsv)
+  - AUC > 0.05 for [`HALLMARK_EPITHELIAL_MESENCHYMAL_TRANSITION`](https://www.gsea-msigdb.org/gsea/msigdb/human/geneset/HALLMARK_EPITHELIAL_MESENCHYMAL_TRANSITION.html)
+- `tumor EWS-high proliferative`:
+  - Cells that met the criteria for `tumor EWS-high` and had mean expression of marker genes for proliferative tumor cells (see [`references/tumor-cell-state-markers.tsv`](./references/tumor-cell-state-markers.tsv)) > 0  
+
+Defining normal cell types: Consensus cell type labels were obtained by running the [`assign-consensus-celltypes.sh` workflow available in the `cell-type-consensus` module](../cell-type-consensus/assign-consensus-celltypes.sh). 
+The consensus cell type label was used for all cells that did not meet the criteria for tumor cells mentioned above. 
+If no consensus cell type is identified and the cell is not a tumor cell, it will have "Unknown" as the label. 
+
+This script requires the processed `SingleCellExperiment` objects for all samples in `SCPCP000015` as input and was run using the following command: 
+
+```sh
+./assign-consensus-celltypes.sh "SCPCP000015"
+```
+
+Note that an alternative approach we used was to build a custom reference containing both tumor cells and normal cell types for use with `SingleR` to assign cell types. 
+The results from this appraoch was concordant with the above outlined approach and is included in the TSV file with the final annotations (see the [results section below](#annotation-results)). 
+See the [full instructions for the `SingleR` workflow below](#aucell-and-singler-annotation-workflow). 
+
+### Annotation results 
+
+See this [notebook](http://htmlpreview.github.io/?https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/main/analyses/cell-type-ewings/exploratory_analysis/08-merged-celltypes.nb.html) for a summary of how the final cell type annotations were assigned and plots that validate these assignments. 
+
+A TSV file containing the final annotations are stored in S3 at: 
+`s3://researcher-211125375652-us-east-2/cell-type-ewings/results/final-annotations/SCPCP000015_celltype-annotations.tsv.gz`
+
+The TSV file contains the following columns: 
```

Comment:
For the ontology columns, I'd also indicate that the tumor labeled are not ontologies but strings starting with `tumor`